compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -437,6 +441,46 @@ void buildAMDAIETransformPassPipeline(OpPassManager &variantPassManager) {
   });
 }
 
+void addAMDAIEObjectFifoLoweringPasses(OpPassManager &passManager) {
+  passManager.addPass(createEraseHALDescriptorTypeFromMemRefPass());
+  passManager.addPass(memref::createFoldMemRefAliasOpsPass());
+  passManager.addPass(createAMDAIEPackToDmaPass());
+  passManager.addPass(xilinx::air::createCopyToDmaPass());
```

Comment:
Dont have to do this in this PR, but it would be good to just have a pass that lowers memcopy to dma because the `air::createCopyToDmaPass` does a lot of other stuff looking for subviews that we (hopefully) dont need.

Reply:
Good point! I created an issue for this: https://github.com/nod-ai/iree-amd-aie/issues/480

Reply:
Yes, exactly